app/components/Header.tsx

```
@@ -0,0 +1,33 @@
+import React from 'react';
+
+const Header = () => {
+  return (
+    <header className="fixed w-full top-0 z-50 flex justify-between items-center p-4 bg-white/95 backdrop-blur-sm shadow-sm">
+      <div className="text-2xl font-bold text-[#6c63ff]">HostelHub</div>
+      <nav className="hidden md:flex space-x-6">
+        <a href="/" className="text-gray-800 hover:text-[#6c63ff] transition-colors">
```

ESLint

[error] Do not use an `<a>` element to navigate to `/`. Use `<Link />` from `next/link` instead. See: https://nextjs.org/docs/messages/no-html-link-for-pages

(@next/next/no-html-link-for-pages)